.github/workflows/e2e_test.yml

```diff
@@ -13,10 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Go
-
-permissions:
-  contents: read
+name: test
```

Comment:
would call this e2e-test instead of test, we already have a test job below

Reply:
This is by design as it groups tests, if we don't have this it shows up as "e2e-test e2e tests"..

![image](https://user-images.githubusercontent.com/7058938/136004168-a32e68fe-99ad-44d0-b899-9aa104ce4efd.png)

Reply:
ahh, i see what you mean! sounds great then!